medusa/providers/torrent/json/t411.py

```diff
@@ -205,6 +216,8 @@ def search(self, search_strings, age=0, ep_obj=None):
         if not self.login():
             return results
 
+        self._get_terms()
```

Comment:
I've already given my review of plex.  I'm awaiting those changes to be addressed before I review further.  Keep this PRs discussion to this PR please.